pkg/docker/details.js

```diff
@@ -208,6 +208,7 @@ PageContainerDetails.prototype = {
 
         $('#container-details-id').text(info.Id);
         $('#container-details-names').text(util.render_container_name(info.Name));
+        $('#container-details-names').prop('title', util.render_container_name(info.Name));
```

Comment:
This is using the black non-transparent in-browser tooltips instead of our bootstrap ones. You need to call `.tooltip()` here. Also, I think this tooltip would be more useful on the overview page, where the name only has a rather narrow column. (Of course it's fine to show it here, too)